### PR TITLE
[WIP]Gives engineers an cheap power drill to start with

### PR DESCRIPTION
**Pull request by improvedname**
Gives engineers a roundstart power drill which makes them look more proffesional.
the cheap powerdrill is just slightly faster then normal tools,
removes the wrench from the toolbelt and adds a t-ray scanner in addition 

:cl: Improvedname
balance: Gives engineering cheap power drills
/:cl:

### CODE REVIEW

File: code/game/objects/items/tools/screwdriver.dm [hunk 1]
```diff
@@ -117,6 +117,11 @@
 	toolspeed = 0.25
 	random_color = FALSE
 
+/obj/item/screwdriver/power/cheap
```
Comment (Poojawa) [inline code/game/objects/items/tools/screwdriver.dm]: You can easily modularize this.

File: code/game/objects/items/storage/belt.dm [hunk 2]
```diff
@@ -75,18 +75,17 @@
 	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","orange"))
 
 /obj/item/storage/belt/utility/full/engi/PopulateContents()
-	new /obj/item/screwdriver(src)
-	new /obj/item/wrench(src)
+	new /obj/item/screwdriver/power/cheap(src)
```
Comment (Poojawa) [inline code/game/objects/items/storage/belt.dm:80]: modularize if possible, make comments if not.

**Review by Poojawa — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/belt.dm (hunk 2 above)
Comment (silicons) [reply code/game/objects/items/storage/belt.dm:80]: . = ..()
new /obj/item/screwdriver/power/cheap(src)
qdel(locate(/obj/item/wrench) in src)

Comment (silicons) [reply code/game/objects/items/storage/belt.dm:80]: messy but probably works

**Comment by Brimcon** [comment]
Drill converts from "Cheap" to normal Hand drill upon swapping to Wrench mode, or atleast, the description reverts.

**Comment by Poojawa** [comment]
Yee, because the wrench mode wasn't given an override.

**Pull request closed without merging**